gateway/utils/syncman/handle_runner.go

```diff
@@ -71,3 +73,31 @@ func (s *Manager) HandleRunnerRequests(admin *admin.Manager) http.HandlerFunc {
 func (s *Manager) GetRunnerAddr() string {
 	return s.runnerAddr
 }
+
+// GetClusterType returns cluster type
+func (s *Manager) GetClusterType(admin *admin.Manager) (string, error) {
+	s.lock.Lock()
+	defer s.lock.Unlock()
+	if s.runnerAddr == "" {
+		return "none", nil
+	}
+
+	token, err := admin.GetInternalAccessToken()
+	if err != nil {
+		logrus.Errorf("error handling forwarding runner request failed to generate internal access token -%v", err)
+		return "", err
+	}
+
+	data := new(model.Response)
+
+	type favContextKey string
+	var k favContextKey
+	ctx := context.WithValue(context.Background(), k, "Go")
+
+	err = s.MakeHTTPRequest(ctx, http.MethodGet, fmt.Sprintf("http://%s/v1/runner/cluster-type", s.runnerAddr), token, "", map[string]interface{}{}, &data)
```

Comment:
```suggestion
	err = s.MakeHTTPRequest(ctx, http.MethodGet, fmt.Sprintf("http://%s/v1/runner/cluster-type", s.runnerAddr), token, "", map[string]interface{}{}, data)
```
No need to pass a pointer as you are generating model.response with new keyword which gives a pointer by default